client/src/components/DeidentifiedText.js

```diff
@@ -11,7 +11,7 @@ export class DeidentifiedText extends React.Component {
   render() {
     let content;
     if (this.props.text === deidentificationStates.EMPTY) {
-      content = "Input a note and de-identify it in the text box on the left...";
+      content = "Press 'de-identify note' on the left to remove PHI from the clinical note on the left...";
```

Comment:
This is too verbose. What about `Add a clinical note on the left and click on De-identify`. From the title of the app, it's already clear that we are removing PHI.